### PR TITLE
Update contract weights

### PR DESCRIPTION
**Pull request by athei**
They weren't updated after recent PRs.

### CODE REVIEW

**Comment by athei** [comment]
/cmd queue -c bench-bot $ pallet dev pallet_contracts

File: frame/contracts/src/weights.rs [hunk 1]
```diff
@@ -166,2104 +166,2196 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
...
 	// Storage: Contracts OwnerInfoOf (r:1 w:1)
+	// Storage: System EventTopics (r:1 w:1)
 	// Storage: Contracts CodeStorage (r:0 w:1)
 	// Storage: Contracts PristineCode (r:0 w:1)
 	fn remove_code() -> Weight {
-		Weight::from_ref_time(30_016_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(1 as u64))
-			.saturating_add(T::DbWeight::get().writes(3 as u64))
+		Weight::from_ref_time(37_011_000 as u64)
+			.saturating_add(T::DbWeight::get().reads(2 as u64))
```
Comment (ggwpez) [inline frame/contracts/src/weights.rs:276]: There are storage count changes, so the weights were definitely out-dated.  
[SWC compare](https://weights.tasty.limo/compare?repo=substrate&threshold=10&path_pattern=frame%2F*%2Fsrc%2Fweights.rs&method=guess-worst&ignore_errors=true&unit=weight&old=master&new=at/contract-weights).

Comment (athei) [reply frame/contracts/src/weights.rs:276]: Yes. This was the PR that added topics to the event.

**Review by agryaznov — APPROVED** [review]

**Review by ggwpez — APPROVED** [review]

**Comment by athei** [comment]
bot merge

**Pull request merged**